src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/reporting/ndmis/performance/NdmisDateTime.kt

```diff
@@ -0,0 +1,14 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.reporting.ndmis.performance
+
+import java.time.OffsetDateTime
+import java.time.ZoneOffset
+import java.time.format.DateTimeFormatter
+
+// Agreement: the v2 NDMIS report uses this format, v3 switches to proper ISO8601
+// Deadline: stop producing v2 format after 13 December 2021
```

Comment:
it didn't sound like the NDMIS team had signed off on this?

Reply:
yeah, not yet. I'll amend the deadline when we agree